pf/internal/schemashim/provider.go

```diff
@@ -16,13 +16,16 @@ package schemashim
 
 import (
 	"context"
-	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
 
 	pfprovider "github.com/hashicorp/terraform-plugin-framework/provider"
+
 	"github.com/pulumi/pulumi-terraform-bridge/pf/internal/pfutils"
+	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim/util"
 )
 
 type SchemaOnlyProvider struct {
+	util.UnimplementedProvider
```

Comment:
This change doesn't seem necessary and it degrades error messages. util.UnimplementedProvider states "for use in testing" and returns "unimpleneted" panics. The panics here were more informative. Can we consider reverting out? Separate refactor PRs to clean things like this up are welcome, we can figure out how to keep the error messages informative with some more work.

Reply:
I added that change because it would have allowed the random downstream test to go green. I can back it out and put it in another PR.